griptape/tools/__init__.py

```diff
@@ -43,5 +46,8 @@
     "TaskMemoryClient",
     "Computer",
     "OpenWeatherClient",
-    "ImageGenerator",
+    "PromptImageGenerator",
+    "ImageVariationGenerator",
+    "ImageInpainter",
+    "ImageOutpainter",
```

Comment:
Similar naming comment. And maybe we should rename to be "Clients"? `InpaintingImageGenerationClient`?